live_photos_test.go

```diff
@@ -0,0 +1,67 @@
+package onfido_test
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/gorilla/mux"
```

Comment:
Can you please run `goimports` on this to separate the github stuff from the standard library things? I will raise an issue to add some linting to the circle build to catch small things like this...